app/dashboard/src/components/AriaComponents/Form/Form.tsx

```diff
@@ -147,30 +158,13 @@ export const Form = React.forwardRef(function Form<
       register: (name, options) => {
         const registered = register(name, options)
 
-        /**
```

Comment:
Are you sure we can now safely remove it? The mapping is essential to support both native and `react-aria` based components.

Reply:
it's not being removed, the function was moved to the top and the wrapping was inlined into the `const result`. can be reverted but i just figured it lets us avoid typingn the same long generic type three times

Reply:
Sorry, missed that 🤦